.github/workflow/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: Resolwe CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+  schedule:
+    - cron: "30 2 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        tox-env: [py36, py37, py38, linters, packaging, migrations]
+        include:
+          - tox-env: py36
+            python-version: 3.6
+          - tox-env: py37
+            python-version: 3.7
+          - tox-env: py38
+            python-version: 3.8
+          - tox-env: linters
+            python-version: 3.8
+          - tox-env: packaging
+            python-version: 3.8
+          - tox-env: migrations
+            python-version: 3.8
+
+    services:
+      postgres: 
+        image: postgres:alpine
+        env:
+          POSTGRES_USER: resolwe
+          POSTGRES_DB: resolwe
+          POSTGRES_PASSWORD: resolwe
+        ports: 
+          - 5432/tcp
+      
+      redis:
+        image: redis:alpine
+        ports: 
+          - 6379/tcp
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip tox
+    - name: Run Tests
+      run: |
+        RESOLWE_DOCKER_DISABLE_SECCOMP=1 tox -r -e ${{ matrix.tox-env }} --skip-missing-interpreters false
```

Comment:
I prefer long flags' names in committed files - `--recreate` instead of `-r`.

Reply:
Also, you can replace `|` in the line above with `>` and break this line into multiple lines as newlines will be ignored by the YAML parser.

Reply:
OK, will try that.